<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="txt_nome_funcionario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_codigo_funcionario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_data_funcionario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>SENHA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cpf_funcionario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_senha_funcionario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>DATA DE NASCIMENTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_voltar_funcionario">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>NOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>CÃ“DIGO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_login_funcionario">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="login_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_cadastrar_funcionario">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CADASTRAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
